lib/index.js

```diff
@@ -208,64 +212,104 @@ module.exports = class Breakpoints {
 
   /**
    * Generate, format and write the CSS file
+   *
+   * @async
    */
-  #generateAndWriteCSS() {
+  async #generateAndWriteCSS() {
     const filePath = path.resolve(process.cwd(), this.#config.css.path);
     let string = "";
     this.#customProperties.forEach((customProperty) => {
       string += `--${customProperty.identifier}: "${customProperty.value}";`;
     });
-    Breakpoints.writeFile(
+    const prettierConfig = {
+      ...(this.#config.prettier?.path
+        ? await Breakpoints.getPrettierConfig(this.#config.prettier?.path)
```

Comment:
The line above already checks for `this.#config.prettier.path` existence, so the optional chaining here is … optional 😁